nin/dasBoot/NodeManager.js

```diff
@@ -5,22 +5,29 @@ class NodeManager {
   }
 
   createNode(nodeInfo) {
-    nodeInfo.options = nodeInfo.options || {};
+    const deepClonedNodeInfo = JSON.parse(JSON.stringify(nodeInfo));
```

Comment:
Is parse . stringify the best way to get a deep copy in JS nowadays 😱

Reply:
It's not the worst, at least.